.github/workflows/docker-run/action.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: common build tasks
```

Comment:
This composite action is the one to be used in https://github.com/elastic/ecs-dotnet/pull/236/checks too